docs/prod-host-setup.md

```diff
@@ -240,252 +240,50 @@ process via the following tools:
-do (grep -q "$cond" /sys/devices/system/cpu/vulnerabilities/l1tf && \
-echo "$cond: ENABLED (OK)") || \
-echo "$cond: DISABLED (Recommendation: ENABLED)"; done
-```
-
-See more details [here](https://www.kernel.org/doc/html/latest/admin-guide/hw-vuln/l1tf.html#guest-mitigation-mechanisms).
-
-##### Apply Speculative Store Bypass (SSBD) mitigation
-
-This will mitigate variants of Spectre side-channel issues such as
-Speculative Store Bypass (Spectre v4) and SpectreNG.
-
-We recommend applying SSBD to Firecracker and the host kernel.
-
-###### X86_64
-
-On x86_64 systems, this can be done using the following kernel cmdline
-parameter:
-
-```console
-spec_store_bypass_disable=on
-```
-
-Unfortunately, this applies SSBD to all the other processes running on the
-host as well.
-
-###### ARM64
-
-On aarch64 systems, SSBD can be applied to the kernel by using the following
-kernel cmdline parameter:
-
-```console
-ssbd=kernel
-```
-
-SSBD is applied to Firecracker by [using the `prctl` interface][3].
-However, this is only available on host kernels Linux >=4.17 and also Amazon
-Linux 4.14. Alternatively, a global mitigation can be enabled by adding the
-following Linux kernel cmdline parameter:
-
-```console
-ssbd=force-on
+grep -q "/dev" /proc/swaps && \
+echo "swap partitions present (Recommendation: no swap)" \
+|| echo "no swap partitions (OK)"
 ```
 
-The following command can be used to check if SSBD is applied to Firecracker:
+### Mitigating hardware vulnerabilities
 
-```bash
-cat /proc/$(pgrep firecracker | head -n1)/status | grep Speculation_Store_Bypass
-```
+> **Note** Firecracker is not able to mitigate host's hardware vulnerabilities.
+Adequate mitigations need to be put in place when configuring the host.
 
-Output shows one of the following:
+> **Note** Firecracker is designed to provide isolation boundaries between
+microVMs running in different Firecracker processes. It is strongly recommended
+that each Firecracker process corresponds to a workload of a single tenant.
 
-- vulnerable
-- not vulnerable
-- thread mitigated
-- thread force mitigated
-- globally mitigated
+> **Note** For security and stability reasons it is highly recommended to load
+updated microcode as soon as possible. Aside from keeping the system firmware
+up-to-date, when the kernel is used to load updated microcode of the CPU this
+should be done as early as possible in the boot process.
 
-##### Hardening other processes
+#### Side channel attacks
 
-For any process running on the host that communicates with Firecracker
-and handles sensitive data, we recommend hardening it against spectre-like
-attacks by:
+It is strongly recommended that users follow the
+[Linux kernel documentation on hardware vulnerabilities](https://docs.kernel.org/admin-guide/hw-vuln/index.html)
+when configuring mitigations against side channel attacks including "Spectre"
+and "Meltdown" attacks
+(see [Page Table Isolation](https://www.kernel.org/doc/html/next/x86/pti.html)
```

Comment:
nit: Are there any reasons why we use the doc on the next tree instead of https://docs.kernel.org/arch/x86/pti.html ?

Reply:
no reason. Fixed, thanks.